### PR TITLE
MixedRealityProjectConfigurator Lambdas instead of anonymous methods

### PR DESCRIPTION
**Pull request by Alexees**
## Overview
MixedRealityProjectConfigurator inhabits a lot of anonymous methods that add a little overhead to the overall readability and lambdas are just shorter and nicer and... you know, lambdas.
This PR converts them all.

### CODE REVIEW

**Comment by julenka** [comment]
Thank you for the contribution. Could you please [link an issue to this pr](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/PullRequests.html#creating-a-pull-request)?

**Comment by Alexees** [comment]
@julenka even though its not?

File: Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs [hunk 1]
```diff
@@ -96,70 +96,63 @@ public bool IsActiveBuildTargetValid()
...
-            { Configurations.LatestScriptingRuntime, () => { SetLatestScriptingRuntime(); } },
-            { Configurations.ForceTextSerialization,  () => { SetForceTextSerialization(); } },
-            { Configurations.VisibleMetaFiles,  () => { SetVisibleMetaFiles(); } },
-            { Configurations.VirtualRealitySupported,  () => { XRSettingsUtilities.LegacyXREnabled = true; } },
-            { Configurations.SinglePassInstancing,  () => { MixedRealityOptimizeUtils.SetSinglePassInstanced(); } },
-            { Configurations.SpatialAwarenessLayer,  () => { SetSpatialAwarenessLayer(); } },
+            { Configurations.LatestScriptingRuntime,SetLatestScriptingRuntime },
+            { Configurations.ForceTextSerialization, SetForceTextSerialization },
+            { Configurations.VisibleMetaFiles, SetVisibleMetaFiles },
+            { Configurations.VirtualRealitySupported, () => XRSettingsUtilities.LegacyXREnabled = false },
```
Comment (keveleigh) [inline Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs]: Why did this one flip from `true` to `false`? It should be enabling the XR setting

Comment (Alexees) [reply Assets/MRTK/Core/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs]: Nice catch, clumsy me

**Pull request merged**